modules/core/src/main/java/org/apache/ignite/events/EventType.java

```diff
@@ -922,6 +923,19 @@
      */
     public static final int EVT_CLUSTER_DEACTIVATED = 141;
 
+    /**
+     * Built-in event type: node validation failed.
+     * <br>
+     * This event is triggered if a node join fails due to a node validation failure.
+     * <p>
+     * NOTE: all types in range <b>from 1 to 1000 are reserved</b> for
+     * internal Ignite events and should not be used by user-defined events.
+     *
+     * @see NodeValidatedEvent
+     * @see GridComponent#validateNode
+     */
+    public static final int EVT_NODE_VALIDATION_FAILED = 150;
```

Comment:
Why not 142?

Reply:
It seems, that for each new event packet ten of the following indexes are reserved. Thus, index 142 is associated with cluster activation events. But it is not.